pkg/api/apiv1/apiv1.go

```diff
@@ -92,6 +92,9 @@ func (a *router) setup() {
 		r.Post("/cancellations", a.createCancellation)
 		r.Get("/cancellations", a.getCancellations)
 		r.Delete("/cancellations/{id}", a.deleteCancellation)
+
+		// NOTE: this is for dev server only, and not an actual v1 API
+		r.Post("/traces", a.OTLPTrace)
```

Comment:
If this is not a V1 API, should we move it to eg /dev/traces out of the v1 package?  The V1 API is specifically meant to be shared between OSS and cloud

Reply:
I can do that.

Reply:
Done